tests/test_version.py

```diff
@@ -1,8 +1,9 @@
 import sys
-import toml
-
 from pathlib import Path
+
 import pytest
+import toml
+
```

Comment:
Please revert the import cleanups from this commit, since they are unrelated to the new converter.